app/services/tinyurl.rb

```diff
@@ -0,0 +1,6 @@
+class TinyURL
+  def self.short(url)
+    HTTParty.get("http://tinyurl.com/api-create.php?url=#{url}")
```

Comment:
This method should rescue any exception(s) that might occur, write something to the log, and return `nil`.

It looks like we can just rescue `HTTParty::Error`, since all the exceptions inherit from that: https://github.com/jnunemaker/httparty/blob/master/lib/httparty/exceptions.rb

Reply:
The "log" that I refer to should be specific to the URL shortening service.

The log should only contain exceptions - no need to log successful execution.

For an example of logging exception, you could look at `log_hips_activity` method in `payments_controller.rb`.  We use our `ActivityLogger` class for this type of logging.